src/components/Checkout/InContext/InContextCheckout.vue

```diff
@@ -69,21 +69,68 @@
 				{{ customCheckoutButtonText }}
 			</kv-button>
 		</div>
-		<div class="in-context-payment-conttrols" v-else>
+		<div class="in-context-payment-controls" v-else>
+			<kv-grid v-if="!continueAsGuest" class="tw-grid-cols-6">
+				<div class="tw-col-start-4 tw-col-end-7 tw-text-right">
+					<kv-button
+						class="smaller checkout-button tw-mb-1"
+						v-if="promoGuestCheckoutEnabled"
+						@click="continueAsGuest = true"
+					>
+						Continue as a guest
+					</kv-button>
+					<kv-button
+						class="smaller checkout-button tw-mb-1"
+						v-if="promoGuestCheckoutEnabled"
+						variant="secondary"
+						:href="registerOrLoginHref"
+					>
+						Continue as an existing user
+					</kv-button>
+				</div>
+				<div class="tw-col-start-4 tw-col-end-7">
+					<p class="tw-grid-row">
+						For existing users, we recommend you log in with your work email address.
```

Comment:
Is this component used in a place where a user might use a non-work-address? Asking because of the callout in this copy.

Reply:
Agree, we should condition this block on `promoGuestCheckoutEnabled`

Reply:
I'll forward this question to Sara from design.  Her designs show this as the copy though.

Reply:
So it sounds like for these cases where the sponsor is not requiring work addresses, we can omit this copy -- we would still need to have a check for this placed somewhere, which we can make sure we add before enabling the feature flag here but I don't think it should block this PR from going in right now.

Reply:
This is what Sara said ^